src/lint/rules/tests/spacing/mod.rs

```diff
@@ -0,0 +1,210 @@
+use crate::lint::rules::tests::common::assert_snippet;
+use crate::lint::rules::instantiate_rules;
+use crate::lint::LintCfg;
+
+// Put whitespace (space or newline):
+//  SP.reserved around reserved words, such as if, else, default,
```

Comment:
Confused by what rule this is (not) testing. Aren't keywords, on account of being parsed, always surrounded by whitespace?
For example: 'for' is a keyword but 'forfoo' isn't

Reply:
Yes, in your example forfoo is not a keywordThis rule is specifically targets keywords surrounded by other tokens. In the example below, `if(` is a keyword right next to a `(` token. But I see where you are coming from. The document just specifies some guidelines but is good to question the wording of the rules. Sometimes its not that clear what they are testing for.

Reply:
Thanks for the explanation. I am actually not-to-familiar with the underlying indentation rules so the intent is not always clear to me.

The follow-up is then; why is the snippet not currently used for any particular test?